hadoop-hdds/common/src/main/resources/ozone-default.xml

```diff
@@ -2478,4 +2478,18 @@
       rules in Amazon S3's object key naming guide.
     </description>
   </property>
+
+  <property>
+    <name>ozone.om.enable.filesystem.paths</name>
+    <tag>OZONE, OM</tag>
+    <value>false</value>
+    <description>If true, OM will normalize the key paths and also create
```

Comment:
I would rephrase first sentence a bit to state that key names will be interpreted as file system paths. `/` will be treated as a special character and paths will be normalized and must follow Unix filesystem path naming conventions.